package.json

```diff
@@ -8,7 +8,7 @@
     "build": "webpack --mode production",
     "client": "webpack-dev-server --mode development --devtool inline-source-map --hot",
     "server": "nodemon ./server/bin/www",
-    "dev": "export NODE_ENV=hot-dev && concurrently \"npm run server\" \"npm run client\"",
+    "dev": "set -o allexport && source hot-dev.env && set +o allexport && concurrently \"npm run server\" \"npm run client\"",
```

Comment:
Are we still supposed to be sourcing hot-dev.env here? Confused if that file is still going to exist, as we've removed it from the .gitignore and changed the README to say development.env.

Reply:
Yes.  We still need to maintain two separate sets of parameters for hot-dev and development, because they are different contexts.  `hot-dev.env` is for when you're running node locally on your host machine, in which case the `host` value for the database is `localhost`.  `development.env` is for when you are running a docker container, in which case the `host` value for the database is `172.17.0.2`, cause of the way docker networking works.

The thing removed from gitignore was the configuration files.  Added back to gitignore was a general line for all .env files (`*.env`), so both `hot-dev.env` and `development.env` should be ignored by git.

Reply:
neato